examples/objectServerExample/src/main/java/io/realm/examples/objectserver/CounterActivity.kt

```diff
@@ -92,16 +93,16 @@ class CounterActivity : AppCompatActivity() {
         user?.let {
 
             // Create a RealmConfiguration for our user
-            val config = user!!.createConfiguration(BuildConfig.REALM_URL)
-                    .initialData { realm -> realm.createObject(CRDTCounter::class.java, it.identity) }
+            val config = it.createConfiguration(BuildConfig.REALM_URL)
```

Comment:
I really think the usage of `it` here makes it hard to read. I think either my initial proposition, or `user?.let { syncUser ->` would be better.

Reply:
Yeah, I could agree with that. In larger code blocks it can be a bit hard to reason about (at a glance) what `it` actually is. A named parameter, in that case, is probably preferable.